### DISCUSSION TITLE
What goes into 8.3 and do we have an estimate of when to release it?

### DISCUSSION THREAD

**Discussion by SilkieSabra** [OpenCollarTeam/OpenCollar#1002] [Q&A]
We need a clearing house for input from our developers to get together on what goes into the next version and when to release it.  We already have some important bug fixes waiting.  Can we list what the main fixes and features will be?

**Comment by Medea-Destiny** [comment]
We've got Yosty's boot speed up and separation of OC_WELD into a separate script (which will require some work removing it from oc_core and oc_settings) and I'm working on reinstating the lost interface channel stuff. These changes will need some solid testing in a release candidate. 
There's phidoux's fixes to OC outfit, Nikki's fixes to the capture timer, lockguard chains and to removing group on runaway, yosty's efficiency run on oc_dialog, ping's meshlabel visibility fix and caps fix in garble, Phuk's badwords owner change fix, my fix for stray restriction, safeword  chat commands and oc_titler

**Reply by NikkiLacrima** [reply]
I have changes to "runaway" and oc_spy that should be in 8.3 , also looked at sorting results for for RLV folder display, not sure if that is high prio

**Comment by SilkieSabra** [comment]
Are there any issues outstanding that could be fixed for 8.3?

**Reply by Medea-Destiny** [reply]
PR #999 there's a situation Ping spotted that needs another once over, very small easy fix

**Comment by Medea-Destiny** [comment]
I'd like to draw people's attention to #944 which could use some input.

**Comment by NikkiLacrima** [comment]
We need to make the process of getting PR's either dropped or tested-fixed-merged into the development brach a bit more smooth.
Having several patches in PR's refering to the same file waiting with "do not merge yet" is not a good situation.

I know I myself can be faster in updating PRs with suggestion, but its a thing for all of us to work on.

We should have a process to either:
* move a PR to a future release ahead of current dev release, now this can only be considered code samples since it is against a futeure code base
* deny the PR
* or set labet to current dev release and start discussing and testing
* * iterate test results and updates and if
* * * * noone test or comments close it for lack of response
* * * * poster does not update in response to tests or comments withing a reasonable time close for lack of maintenance
* * Once tested and commented and updated should be merged, since it would seem no reason NOT to merge 

This is not very complex scheme , the new is that we do not allow PR's to be against current dev branch to hang around for long without being fixed for merging

/Nikki
